man/systemd.exec.xml

```diff
@@ -2375,6 +2375,36 @@ RestrictNamespaces=~cgroup net</programlisting>
         <xi:include href="version-info.xml" xpointer="v233"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DelegateNamespaces=</varname></term>
+
+        <listitem><para>Delegates ownership of the given namespaces to the processes of this unit. For details
```

Comment:
well, we cannot delegate to processes, only to user namespaces. I think this should be made clear here. i.e. say "to the user namespace of the processeses of the unit"